changelogs/fragments/124-airbrake_deployments-api_v4_for_deploy_notices.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+- "airbrake_deployment - Allow deploy notifications for Airbrake compatible v2 api (e.g. Errbit)"
+
+minor_changes:
+- "airbrake_deployment - Add deprecation notice for ``token`` parameter and v2 api deploys"
```

Comment:
This should be in a `deprecated_features:` section. Please also add that this feature will be removed in `Ansible 2.14`. (We'll eventually adjust that string, but right now it's good enough for search+replace :) )

Reply:
done